DisplaySettings/DisplaySettings.cpp

```diff
@@ -773,15 +772,12 @@ namespace WPEFramework {
             }
 
 		    if(hdmiin_hotplug_port == hdmiArcPortId) { //HDMI ARC/eARC Port Handling
-			bool arc_port_enabled =  false;
 
-                        JsonObject audioOutputPortConfig = DisplaySettings::_instance->getAudioOutputPortConfig();
+                	JsonObject audioOutputPortConfig = DisplaySettings::_instance->getAudioOutputPortConfig();
 			if (audioOutputPortConfig.HasLabel("HDMI_ARC")) {
                             try {
-                                    arc_port_enabled = audioOutputPortConfig["HDMI_ARC"].Boolean();
                             }catch (const device::Exception& err) {
                                     LOGERR("HDMI_ARC not in config object \n");
-                                    arc_port_enabled = false;
```

Comment:
you remove unused var, which is good, now it's an empty try-catch. Shouldn't you also make sure the code makes sense after your fixes?

Reply:
I'll remove the try/catch and the "audioOutputPortConfig", since it is not used anywhere. I am surprised as this is not flagged as unreachable code by compiler.